### PR TITLE
Add support for pre-2016 (non-JSON) Monero wallet keys files

### PR DESCRIPTION
**Pull request by patrickd-**
This improves https://github.com/openwall/john/pull/3151

### CODE REVIEW

File: src/monero_fmt_plug.c [hunk 1]
```diff
@@ -62,7 +62,9 @@ static struct fmt_tests tests[] = {
...
+	// bitmonero wallet v0.8.6.295(0.1-g5ceffa8) (June, 2014)
+  {"$monero$0*982cc13dd90ed2e2970294b2a67480826acaf8eeedec74acccc5929c33d6b7cf01609f0ddb72817579c4375bd1d8f4d47c36c67d83bd3a817d1242211c6824720e430757c24238189875fbb7fe5e643b4273ac3912c7f4f6998ed1086634cebcce035a4af0122c2a6e601ff160bcfa0cfce06d8b3dd38915aa4f6067a732f954da2029ccca5664bbf5df2b2ce046918eec9b71224ec75418686eb437678ebfba50ed3e07ce0bba39f45610775b9958f44b3a53bf3294d7c55f2c455e2ada17f4426969fbc57cc659793892e558b163dd4826889c865b685ea49c54744ecbfcce9d9d39d16e8874a35ec767ab5b69e498f0989f7dd22becb2d50797e7c2867536bc6a1e9b0ecc0d864204c5114fef9ed86d962fced5cac90dbb33a72d16753075bf", "test"},
```
Comment (magnumripper) [inline src/monero_fmt_plug.c]: Indentation should be one tab, no spaces.

**Comment by solardiz** [comment]
Thank you, @patrickd-! When fixing the indentation issue, can you also add a `NEWS` entry please (to the end of the new section for changes after 1.9.0-jumbo-1).

Also, can you please add a sample wallet file (without a balance and with a known password), or several files, via a PR to https://github.com/openwall/john-samples

**Comment by solardiz** [comment]
> [Followup to PR](/openwall/john/pull/4732/commits/b054c4001a0fdce2ed1be94e9ca7af91065efb11) #4732[: Fixed indentation](/openwall/john/pull/4732/commits/b054c4001a0fdce2ed1be94e9ca7af91065efb11)

Please be amending and force-pushing your one commit instead of adding more commits. Thanks!

**Comment by patrickd-** [comment]
Thanks! All done, I think

File: doc/NEWS [hunk 2]
```diff
@@ -186,6 +186,8 @@ Major changes from 1.9.0-jumbo-1 (May 2019) in this bleeding-edge version:
 - Drop support for Apple's CommonCrypto, it was defunct for many years anyway.
   [magnum; 2021]
 
+- Improved Monero support: Now supports legacy wallets from before 2016 that are
+  not using JSON format yet. [patrickd, magnumripper, solardiz; 2021]
```
Comment (solardiz) [inline doc/NEWS:190]: This should be credited to patrickd only.

**Review by solardiz — APPROVED** [review]

File: doc/NEWS (hunk 2 above)
Comment (solardiz) [reply doc/NEWS:190]: I'll merge now, and drop the extraneous credits when going over the `NEWS` file prior to the release.

Comment (patrickd-) [reply doc/NEWS:190]: Ah ok, in other projects everyone participating in the issue is credited. This might be something to add to the CONTRIBUTE file

Comment (magnumripper) [reply doc/NEWS:190]: Fixed in #4735 while at it

**Pull request merged**